bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java

```diff
@@ -102,7 +102,7 @@ public void open() throws Exception {
           CommPortIdentifier.getPortIdentifier(m_Parameters.getPortName());
     } catch (NoSuchPortException e) {
       if(Modbus.debug) System.out.println(e.getMessage());
```

Comment:
Hi @teichsta , there's a huge load of System.out logging in the modbus binding, changing one seems like a drop in a bucket and I'm leaving for vacation today.

My offer: I'm going to change a bunch of them in a couple of weeks, when I'm back.